incl/footer.html

```diff
@@ -12,6 +12,9 @@
     <div id="block-aggregator-feed-1" class="clear-block block block-aggregator">
       <h2 class="title">Recent news</h2>
       <div style="background-color: blanchedalmond; padding-left: 5px;" class="content">
+        <p>October 2023: we have decided to rename 'Coq' into 'The Rocq Prover'. The rename is currently in preparation, and will hopefully happen sometime in 2024.</p>
```

Comment:
Here's my suggestion to provide (hopefully) just the right amount of information:

```suggestion
        <p>The Coq team has decided that Coq will be renamed into 'The Rocq Prover'. Background information available <a href="https://coq.discourse.group/t/coq-community-survey-2022-results-part-iv-and-itp-paper-announcement/2001#renaming-coq-8">here</a>. The rename is currently in preparation, we hope to have a new visual identity and website by the end of 2024, and to do a first release of Rocq around that time.</p>
```

Reply:
Great, thanks! I updated the PR.